packages/lesswrong/components/sunshineDashboard/SunshineNewUsersInfo.tsx

```diff
@@ -126,6 +126,19 @@ const SunshineNewUsersInfo = ({ user, classes, updateUser }: {
 
   const canReview = !!(user.maxCommentCount || user.maxPostCount)
 
+  const handleNotes = () => {
+    updateUser({
+      selector: {_id: user._id},
+      data: {
+        sunshineNotes: notes
+      }
+    })
+  }
+
+  useEffect(() => {
+    handleNotes()
```

Comment:
I think I remember a conversation about this, where the intent was that it saves on close. But I think this is saving on open instead, which is not what you want.

Reply:
Nvm, was reading commit-by-commit

Reply:
My PRs are almost never optimized for that, alas

Reply:
I had tested the previous version and it seemed to work (I think once in the past day it failed and I attributed it to "idk some race condition that was sort of fine"). 

But, the new version also seems to work, and if you have a model that says it's supposed to work better, seems good.